tests/cmd/logout/out/$HOME/.buffrs/credentials.toml

```diff
@@ -0,0 +1 @@
+credentials = []
```

Comment:
I think this should be completely empty in the best case, i think serde can to that with `skip`

Reply:
Also a nit (good become a "good first issue")

Reply:
It's easy to fix. Do you think it should be left as a good first issue, or do I just fix it? I don't mind either way.

Reply:
Up to u :)